src/main/java/com/sap/sgs/phosphor/fosstars/model/score/oss/ProjectSecurityAwarenessScore.java

```diff
@@ -54,6 +61,8 @@ public ScoreValue calculate(Value... values) {
         "Hey! You have to tell me if the project has a security policy!");
     Value<Boolean> securityTeam = findValue(values, HAS_SECURITY_TEAM,
         "Hey! You have to tell me if the project has a security team!");
+    Value<Boolean> signedCommits = findValue(values, USES_VERIFIED_SIGNED_COMMITS,
+        "Hey! You have to tell me if the project uses any signed commits!");
```

Comment:
Please update it to "Hey! You have to tell me if the project uses verified signed commits!" according to the feature name.